### PR TITLE
Move Math API to separate page

### PR DESCRIPTION
**Pull request by neon60**
Move Math API to separate page

### CODE REVIEW

File: docs/reference/math_api.rst [hunk 1]
```diff
@@ -0,0 +1,1121 @@
+.. meta::
+  :description: This chapter describes the built-in math functions that are accessible in HIP. 
+  :keywords: AMD, ROCm, HIP, CUDA, math functions, HIP math functions
+
+.. _math_api_reference:
+
+********************************************************************************
+HIP MATH API
```
Comment (lpaoletti) [inline docs/reference/math_api.rst]:
```suggestion
HIP math API
```

File: docs/reference/math_api.rst [hunk 2]
```diff
@@ -0,0 +1,1121 @@
...
+********************************************************************************
+
+HIP-Clang supports a set of math operations that are callable from the device. HIP supports most of the device functions supported by CUDA. These are described in the following sections.
```
Comment (lpaoletti) [inline docs/reference/math_api.rst]:
```suggestion
HIP-Clang supports a set of math operations that are callable from the device. HIP supports most of the device functions supported by NVIDIA CUDA. These are described in the following sections.
```

File: docs/reference/math_api.rst (hunk 1 above)
Comment (lpaoletti) [reply docs/reference/math_api.rst]: This change should also be made here:
image

Comment (lpaoletti) [reply docs/reference/math_api.rst]: and here:
image

File: docs/reference/math_api.rst (hunk 2 above)
Comment (lpaoletti) [reply docs/reference/math_api.rst]: Do you need to note anything about including in header files?

Comment (lpaoletti) [reply docs/reference/math_api.rst]: Suggestion:
Perhaps HIP-Clang should be a hyperlink to the relevant info?

File: docs/reference/math_api.rst (hunk 1 above)
Comment (neon60) [reply docs/reference/math_api.rst]: Fixed the title

**Comment by neon60** [comment]
@randyh62 @lpaoletti please review and approve, if everything is fine with this.

**Review by randyh62 — APPROVED** [review]

**Pull request merged**